### PR TITLE
Digitalocean ex_rename_node() extension method

### PR DESCRIPTION
**Pull request by fluffybeing**
@Kami as you have told me; I have added the extension method for renaming  digitalocean node and also test suit for it.

### CODE REVIEW

**Comment by Kami** [comment]
This looks like a good start.

There are some lint and test failure issues though - https://travis-ci.org/apache/libcloud/builds/19144330

For tests to pass you need to fix the method call, add a response fixture and modify the MockHttpClass to return your response fixture on rename node HTTP request.

File: libcloud/compute/drivers/digitalocean.py [hunk 1]
```diff
@@ -127,6 +127,12 @@ def destroy_node(self, node):
                                       params=params)
         return res.status == httplib.OK
 
+    def ex_rename(self, node, name):
```
Comment (Kami) [inline libcloud/compute/drivers/digitalocean.py]: Please add `_node` suffix to the method name to make it clear that the method operates on a node.

File: libcloud/test/compute/test_digitalocean.py [hunk 2]
```diff
@@ -89,6 +89,11 @@ def test_destroy_node_success(self):
         result = self.driver.destroy_node(node)
         self.assertTrue(result)
 
+    def test_ex_rename_node_success(self):
+        node = self.driver.list_nodes()[0]
+        result = self.driver.ex_rename_node(node, "helios")
```
Comment (Kami) [inline libcloud/test/compute/test_digitalocean.py]: For consistency, please use single quotes around strings.

**Comment by fluffybeing** [comment]
@Kami  Thanks  I will make the changes :)

**Comment by fluffybeing** [comment]
@Kami done :+1:

**Comment by Kami** [comment]
@rahulrrixe Looking good.

Please just squash all the commits (https://libcloud.readthedocs.org/en/latest/development.html#squash-the-commits-and-generate-the-patch) and I will merge changes into trunk.

**Comment by Kami** [comment]
Merged into trunk, thanks.

This pull request can be closed now.

**Comment by fluffybeing** [comment]
@Kami  Thanks :+1:  looking forward for next task.

**Pull request closed without merging**